source/2-our-operating-process.md

```diff
@@ -1,12 +1,12 @@
 # Our Operating Process
 
 ## Iterations and Hours
-We work on client projects in weekly iterations, four days per week. We typically work Monday through Thursday (barring no major holidays or staff illness). Fridays are normally reserved for echobind’s internal business tasks. We work during normal business hours, but do not track the exact number of hours worked. All work that is completed during the week is clearly documented via source control, project management systems, chat and weekly recap meetings.
+We work on client projects in weekly iterations, four days per week. We typically work Monday through Thursday (barring no major holidays or staff illness). Fridays are normally reserved for echobind's internal business tasks. We work during normal business hours, but do not track the exact number of hours worked. All work that is completed during the week is clearly documented via source control, project management systems, chat and weekly recap meetings.
```

Comment:
Do we want to recommend any readings on why we do this? I know personally "Rework", "Four Day Workweek" and other sources might be helpful here for those not familiar with the concept.